src/main/java/org/jabref/gui/search/GlobalSearchResultDialog.fxml

```diff
@@ -15,9 +14,6 @@
         <VBox>
             <HBox fx:id="searchBarContainer" spacing="10" alignment="CENTER_RIGHT">
                 <ToggleButton fx:id="keepOnTop" styleClass="icon-button,narrow" prefHeight="20.0" prefWidth="20.0">
-                    <graphic>
```

Comment:
Why was this icon removed?

Reply:
> Why was this icon removed?

It is unnecessary since we have a subscription in the `initialize()` method:
```
keepOnTopSubscription = EasyBind.subscribe(viewModel.keepOnTop(), value -> {
    keepOnTop.setGraphic(value
            ? IconTheme.JabRefIcons.KEEP_ON_TOP.getGraphicNode()
            : IconTheme.JabRefIcons.KEEP_ON_TOP_OFF.getGraphicNode());
});
```